demosplan/DemosPlanCoreBundle/Resources/config/GlobalConfig.php

```diff
@@ -818,7 +818,16 @@ public function setParams(ParameterBagInterface $parameterBag, TranslatorInterfa
 
         $this->advancedSupport = $parameterBag->get('advanced_support');
 
-        $this->externalLinks = $this->getValidatedExternalLinks($parameterBag);
+        $externalLinks = $parameterBag->get('external_links');
+        if (is_array($externalLinks)) {
+            foreach ($externalLinks as &$externalLinkUrl) {
```

Comment:
imho using a reference here is quite error prone. If you insist we can discuss this, but we use variable references only in rare cases for (in my eyes) very good reasons to increase maintainability.